Added test for BiblioEditCommand

DoItTomorrow opened the pull request:
Updated Default sources to have more fields that are filled.
Updated Sample sources to match test TypicalSources

DoItTomorrow committed:
Merge pull request #7 from CS2103-AY1819S2-W13-3/master

Update merge from 13-3/main

Commit:
Added test for BiblioEditCommand

DoItTomorrow committed:
Merge branch 'master' into master

Commit:
Updated SampleDataUtil to match test data.

Commit:
Checkstyle Edits

DoItTomorrow committed:
Merge branch 'master' into master

Commit:
Merge branch 'master' of https://github.com/DoItTomorrow/main

ToonDragon commented:
Looking at the failed tests, it seems like the typical sources have not been updated appropriately.  Perhaps you can check if the biblio fields in the typical sources have been accounted for in the .equals() override command (I had a similar problem for author but rectified it relatively easily).

Can also check the typical sources file itself and see if the bibilo fields have been entered accurately.  If they are not correct then the parser will not work properly and cause it to fail.

Commit:
Removed old constructors for Source.

DoItTomorrow committed:
Merge branch 'master' into master

Commit:
Ignored 2 tests that are failling when run on their own but not when runned with all other tests

Commit:
Revert "Ignored 2 tests that are failling when run on their own but not when runned with all other tests"

This reverts commit 486159fda70bc7c5f700612a977786840ee2bfe2.

Commit:
Ignored 2 tests
Tests are passing when run on their own but not when runned with all other tests

Commit:
Removed unnecessary toString in EditCommand

Commit:
Checkstyle Edit

DoItTomorrow committed:
Merge branch 'master' into master

Commit:
Update to BiblioEditCommand to not break other tests. One test is current ignored.

Commit:
Checkstyle Edits

Commit:
Checkstyle Edits

ToonDragon reviewed (approved):
Additional test functionality for the biblio edit command.  One of the test is set to @Ignore due to unforeseen circumstances involving test interaction.  

Approved for now but will need to see if the test can be rectified in the future (by future i mean tomorrow).

Commit:
Added test for SampleDataUtil

DoItTomorrow committed:
Merge branch 'master' into master

Commit:
Removed unnecessary import in util/TypicalSources

Commit:
Merge branch 'master' of https://github.com/DoItTomorrow/main

Commit:
Checkstyle Edits

Commit:
Removed unnecessay import in util/TypicalSources.java

Commit:
Readded accidental removal of import
